<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Innio Security</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QMainWindow{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>531</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="videoBtn">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>270</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/videoIcon.png</normaloff>img/videoIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="photoBtn">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/photoIcon.png</normaloff>img/photoIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
